pyproject.toml

```diff
@@ -15,6 +15,8 @@ dependencies = [
     "ipykernel>=6.29.5",
     "numpy>=2.2.6",
     "pandas>=2.3.0",
+    "jupyter>=1.1.1",
+    "notebook>=7.4.3",
```

Comment:
these should be moved to dependency-groups > dev (ipykernel from above can be removed)